roles/openshift_logging_curator/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ### General logging settings
 openshift_logging_curator_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_curator_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_curator_image_version: "{{ openshift_logging_image_version }}"
```

Comment:
Even though `openshift_logging_{{role}}_image_version` vars were never used anywhere? These vars were only defined at one point, but not used. 

Without basing the `openshift_logging_{{role}}_image_version` on the common `openshift_logging_image_version`, I am not sure I understand how to draw the relationship there because if:

- we revert these and keep the rest of the PR - then the user has to define `openshift_hosted_logging_deployer_version` to change the version installed for all roles at once, `openshift_logging_image_version` does nothing, but the user can have fine control over each role image version
- we revert these and invert the relationship between `openshift_logging_image_version` and `openshift_logging_{{role}}_image_version` - all versions are controlled by `openshift_logging_image_version`, we are back to what we had before this PR only slightly more convoluted and confusing

If we can't redefine `openshift_logging_{{role}}_image_version` vars, then I would recommend closing this PR without merge and wait for the 3.8